src/rulesets/Base/array.jl

```diff
@@ -262,19 +262,57 @@ function rrule(::typeof(hvcat), rows, values::Union{AbstractArray, Number}...)
 end
 
 #####
-##### `fill`
+##### `reverse`
 #####
 
-function rrule(::typeof(fill), value::Any, dims::Tuple{Vararg{Int}})
-    function fill_pullback(Ȳ)
-        return (NoTangent(), sum(Ȳ), NoTangent())
-    end
-    return fill(value, dims), fill_pullback
+# 1-dim case allows start/stop, N-dim case takes dims keyword, whose defaults changes in Julia 1.6.
+
+# function frule((_, xdot, _, _), ::typeof(reverse), x::AbstractArray, args...; kw...)
+#     return reverse(x, args...; kw...), reverse(xdot, args...; kw...) # seems not to work
+# end
```

Comment:
do we want to keep this comment?

Reply:
Not really. But perhaps you know why it doesn't work? Keeping just this would be simpler & better match the `rrule`.

IIRC there were difficulties with vararg rrules at some point, maybe this is a bug? Or maybe I did something wrong?